From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-page-table-check-fix

export page_table_check_disabled and __page_table_check_pte_set to modules

ERROR: modpost: "page_table_check_disabled" [drivers/gpu/drm/i915/i915.ko] undefined!
ERROR: modpost: "__page_table_check_pte_set" [drivers/gpu/drm/i915/i915.ko] undefined!

Cc: Pasha Tatashin <pasha.tatashin@soleen.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/page_table_check.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/mm/page_table_check.c~mm-page-table-check-fix
+++ a/mm/page_table_check.c
@@ -19,6 +19,7 @@ static bool __page_table_check_enabled _
 				IS_ENABLED(CONFIG_PAGE_TABLE_CHECK_ENFORCED);
 
 DEFINE_STATIC_KEY_TRUE_RO(page_table_check_disabled);
+EXPORT_SYMBOL(page_table_check_disabled);
 
 static int __init early_page_table_check_param(char *buf)
 {
@@ -217,6 +218,7 @@ void __page_table_check_pte_set(struct m
 				     pte_write(pte));
 	}
 }
+EXPORT_SYMBOL(__page_table_check_pte_set);
 
 void __page_table_check_pmd_set(struct mm_struct *mm, unsigned long addr,
 				pmd_t *pmdp, pmd_t pmd)
_
